multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -13,19 +13,39 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
 
     - name: kubelet.service
       enable: true
       command: start
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=docker.service load-rkt-stage1.service
+        After=docker.service load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+        --mount volume=dns,target=/etc/resolv.conf \
+        --volume=mp,kind=host,source=/usr/sbin/modprobe \
```

Comment:
I believe this is necessary so kubelet can resolv dns based on host settings correctly.  rkt pods launched with kubernetes and host networking will now automatically do this: https://github.com/kubernetes/kubernetes/pull/29024

This won't affect our kubelet-wrapper container though so we do it manually. I think rkt has some kinda `--dns` option too now, there is has been quite a few recent changes in regards to dns in a pod and resolv.conf but it is not clear if we really need to do anything different here.